code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -1183,13 +1183,11 @@
   []
   (let [tr                (subscribe [::i18n-subs/tr])
         nuvlabox          (subscribe [::subs/nuvlabox])
-        nuvlabox-status   (subscribe [::subs/nuvlabox-status])
         zoom              (atom 3)
-        new-location      (r/atom nil)
-        {:keys [id location]} nuvlabox
-        inferred-location (:inferred-location @nuvlabox-status)]
+        new-location      (r/atom nil)]
     (fn []
-      (let [update-new-location #(reset! new-location %)
+      (let [{:keys [id location inferred-location]} @nuvlabox
```

Comment:
`inferred-location` is not in nuvlabox, but nuvlabox-status

Reply:
inferred-location was denormalized to nuvlabox resource sometimes ago

Reply:
then perhaps rename the variable, cause it then get hard to read and inconsistent with the resource name